### PR TITLE
Add a link to the HTML documentation from the main page

### PR DESCRIPTION
**Pull request by xadupre**
### Description
[onnx.ai/onnx](https://onnx.ai/onnx/] is not referenced. This PR adds a couple of links to it from the main README.md file.

### Motivation and Context
Python documentation is automated and much easier to use than the markdown documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,14 +15,16 @@ data types. Currently we focus on the capabilities needed for inferencing (scori
...
 * [Tutorials for creating ONNX models](https://github.com/onnx/tutorials).
 * [Pre-trained ONNX models](https://github.com/onnx/models)
 
 # Learn about the ONNX spec
 * [Overview](docs/Overview.md)
 * [ONNX intermediate representation spec](docs/IR.md)
 * [Versioning principles of the spec](docs/Versioning.md)
-* [Operators documentation](docs/Operators.md)
+* [Operators documentation](docs/Operators.md) (development version)
+* [Operators documentation](https://onnx.ai/onnx/onnx_doc_folder/index.html) (latest release)
```
Comment (jcwchen) [inline README.md:27]: I thought the content in https://onnx.ai/onnx/ will be updated after every update in the main branch?

Comment (xadupre) [reply README.md:27]: I think the update is manual. So it is really up to us.

Comment (gramalingam) [reply README.md:27]: Someone else was asking me a similar question (about `docs/Operators.md`) ... It is a good idea to have a latest-release documentation. (We could add a development version also, if it is feasible to have both ... I mean in `onnx.ai`.)

**Review by gramalingam — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jcwchen) [reply README.md:27]: The [document from website](https://onnx.ai/onnx/onnx_doc_folder/text_diff_ScatterND_16_18.html) seems to include Rama's recent update https://github.com/onnx/onnx/pull/4575. (which is merged after [this PR for page creating](https://github.com/onnx/onnx/pull/4576)) Unless someone is manually triggering it after Rama's PR?

File: README.md [hunk 2]
```diff
@@ -15,14 +15,16 @@ data types. Currently we focus on the capabilities needed for inferencing (scori
 ONNX is [widely supported](http://onnx.ai/supported-tools) and can be found in many frameworks, tools, and hardware. Enabling interoperability between different frameworks and streamlining the path from research to production helps increase the speed of innovation in the AI community. We invite the community to join us and further evolve ONNX.
 
 # Use ONNX
+* [Documentation of ONNX Python Package](https://github.com/onnx/onnx)
```
[the remark this answers is not in the document]
Comment (xadupre) [reply README.md:18]: For new users of onnx, markdown pages are very difficult to use. They is no title, no way to search them, markdown is very slow to render...

File: README.md (hunk 1 above)
Comment (jcwchen) [reply README.md:27]: To make it only trigger manually, we should remove this event https://github.com/onnx/onnx/blob/main/.github/workflows/pages.yml#L7.

Comment (jcwchen) [reply README.md:27]: > To make it only trigger manually, we should remove this event

I have a fresh PR to do that: https://github.com/onnx/onnx/pull/4597

**Pull request merged**